aspnetcore/host-and-deploy/iis/advanced.md

```diff
@@ -388,7 +388,7 @@ The files can be found by searching for `aspnetcore` in the `applicationHost.con
 
 ## Install Web Deploy when publishing with Visual Studio
 
-When deploying apps to servers with [Web Deploy](/iis/install/installing-publishing-technologies/installing-and-configuring-web-deploy-on-iis-80-or-later), install the latest version of Web Deploy on the server. To install Web Deploy, obtain an installer from the [Microsoft Download Center](https://www.microsoft.com/download/details.aspx?id=43717).
+When deploying apps to servers with [Web Deploy](/iis/install/installing-publishing-technologies/installing-and-configuring-web-deploy-on-iis-80-or-later), install the latest version of Web Deploy on the server. To install Web Deploy, use the [Web Platform Installer (WebPI)](https://www.microsoft.com/web/downloads/platform.aspx), which is the preferred installation procedure. WebPI offers a standalone setup and a configuration for hosting providers.
```

Comment:
This new link that has been added will take folks to the download page which doesn't have the download there anymore since it was retired last year.  That page in turn links to this blog: https://blogs.iis.net/iisteam/web-platform-installer-end-of-support-feed which in turn at the very bottom makes the only suggestion we have available:
"Take a look at creating a [WebPI Offline Feed](https://docs.microsoft.com/en-us/iis/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release) - you can setup a mini WebPI feed and installer repository locally that will allow you to continue to deploy products to legacy machines."

The end of the journey would be here:
[Web Platform Installer v5.1 Command Line (WebPICMD.exe) - RTW release](https://learn.microsoft.com/en-us/iis/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release)

Maybe we should just go with the end result link of that path but note up front it is returned but they can still consider that last option.  Save folks the long path to get there and set expectations up front.